iphone/Classes/TiUIView.m

```diff
@@ -1375,6 +1375,14 @@ - (void)processTouchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event
 	}
 }
 
+- (void)processKeyPressed:(id)key
+{
+    if ([self.proxy _hasListeners:@"keyPressed"]) {
+        NSDictionary *evt = [NSDictionary dictionaryWithObjectsAndKeys:[TiUtils stringValue:key],@"keyCode", nil];
+        [self.proxy fireEvent:@"keyPressed" withObject:evt];
```

Comment:
Looks fine! In the last months, we started doing the short-hand implementation of events like this:
```objc
[self.proxy fireEvent:@"keyPressed" withObject:@{@"keyCode": [TiUtils stringValue:key]}];
```